scripts/exp/remove-exp.js

```diff
@@ -32,7 +32,6 @@ const {
 // can be used in command line
 if (argv._[0]) {
   const dirOrFile = path.resolve(argv._[0]);
-
```

Comment:
Doesn't seem necessary?

Reply:
nop, but I don't want to trigger CI again. :)